cnd/src/network/halight.rs

```diff
@@ -0,0 +1,35 @@
+use crate::swap_protocols::{
```

Comment:
:question: Maybe a dumb question, but do all these files that orchestrate a swap need to live in the `network` module? None of them seem to care about being called by the network later.

Reply:
Nice observation, these are literally just a wrapper around `create_watcher` that adds tracing.  I didn't even notice that while doing this.  Adding that to my list also, I'll do a follow up PR.  thanks